src/main/java/com/shaft/cucumber/ElementSteps.java

```diff
@@ -15,11 +13,40 @@
     public ElementSteps(ThreadLocal<WebDriver> driver) {
         if (driver == null) {
             this.driver = new ThreadLocal<>();
-        }else {
+        } else {
             this.driver = driver;
         }
     }
 
+    protected static By getLocatorFromTypeAndValue(LocatorType type, String locatorValue) {
```

Comment:
Codacy found an issue: [The method 'getLocatorFromTypeAndValue(LocatorType, String)' has an NPath complexity of 2147483647](https://app.codacy.com/manual/mohab.mohieeldeen/SHAFT_ENGINE/pullRequest?prid=5771854)